localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -224,9 +230,48 @@
         <trans-unit id="msgPromptRetryConnectionDifferentCredentials">
             <source xml:lang="en">Error: Login failed. Retry using different credentials?</source>
         </trans-unit>
+        <trans-unit id="msgPromptRetryFirewallRuleExtNotInstalled">
+            <source xml:lang="en">Your client IP address does not have access to the server. Download and install the Azure Account extension to sign in to an Azure account and create a new firewall rule to enable access.</source>
+        </trans-unit>
+        <trans-unit id="msgPromptRetryFirewallRuleNotSignedIn">
+            <source xml:lang="en">Your client IP address does not have access to the server. Sign in to an Azure account and create a new firewall rule to enable access.</source>
+        </trans-unit>
+        <trans-unit id="msgPromptRetryFirewallRuleNotActivated">
+            <source xml:lang="en">Your client IP address does not have access to the server. Activate the Azure Account extension and sign in to an Azure account and create a new firewall rule to enable access.</source>
+        </trans-unit>
+        <trans-unit id="msgPromptRetryFirewallRuleSignedIn">
+            <source xml:lang="en">Account signed In. Create new firewall rule? </source>
+        </trans-unit>
+        <trans-unit id="msgPromptRetryFirewallRuleAdded">
+            <source xml:lang="en">Firewall rule successfully added. Retry profile creation? </source>
+        </trans-unit>
+        <trans-unit id="msgPromptAzureExtensionActivatedNotSignedIn">
+            <source xml:lang="en">Azure Account extension activated. Sign in to an Azure account? </source>
+        </trans-unit>
+        <trans-unit id="msgPromptAzureExtensionActivatedSignedIn">
+            <source xml:lang="en">Azure Account extension activated and signed in.</source>
+        </trans-unit>
+        <trans-unit id="msgInvalidIpAddress">
+            <source xml:lang="en">Invalid IP Address </source>
+        </trans-unit>
+        <trans-unit id="msgInvalidEndIpAddress">
+            <source xml:lang="en">Invalid IP Address </source>
+        </trans-unit>
```

Comment:
Is this purposely a duplicate of the one above? I'd just have a single string if so - no reason to duplicate it

Reply:
Whoops, good catch